src/argus/incident/ticket/dummy.py

```diff
@@ -94,11 +97,14 @@ def create_ticket(cls, serialized_incident: dict):
         endpoint, authentication, ticket_information = cls.import_settings()
 
         client = cls.create_client(endpoint=endpoint, authentication=authentication)
-        ticket_url = client.create(
+        client.create(
             {
                 "title": serialized_incident["description"],
                 "description": serialized_incident["description"],
             }
         )
-
-        return ticket_url
+        url = urljoin(
+            getattr(settings, "FRONTEND_URL", ""),
+            reverse("htmx:incident-detail", kwargs={"pk": serialized_incident["pk"]}),
+        )
```

Comment:
This returns the link to the incident detail page, not the fake ticket system page


```suggestion
        url = reverse("htmx:fake-ticket", kwargs={"pk": serialized_incident["pk"]})
```